lang/en.js

```diff
@@ -541,7 +541,10 @@ export default {
     "settings": "Settings",
     "likes": "Likes",
     "publicCollections": "Public Collections",
-    "privateCollections": "Private Collections"
+    "privateCollections": "Private Collections",
+    "likes-empty": "You haven’t liked any items yet",
```

Comment:
Restructure so that the keys are:
* `account.notifications.noPrivateCollections`
* `account.notifications.noPublicCollections`
* `account.notifications.noLikedItems`